internal/pkg/build/sources/conveyorPacker_oci.go

```diff
@@ -235,7 +236,14 @@ func (cp *OCIConveyorPacker) insertOCIConfig() error {
 }
 
 func (cp *OCIConveyorPacker) unpackTmpfs(ctx context.Context) error {
-	imageSource, err := cp.srcRef.NewImageSource(ctx, cp.sysCtx)
+	var sysCtx *types.SystemContext
+	if cp.transportOptions != nil {
```

Comment:
I'm confused about something: why not use ocitransport.SystemContextFromTransportOptions() here, as in line 215? (I think it even includes the same `nil` check that is done explicitly here.)